engine/menus/empty_sram.asm

```diff
@@ -1,12 +1,8 @@
 EmptyAllSRAMBanks:
-	ld a, 0
-	call .EmptyBank
-	ld a, 1
-	call .EmptyBank
-	ld a, 2
-	call .EmptyBank
-	ld a, 3
+FOR BANKNO, NUM_SRAM_BANKS
```

Comment:
STYLE.md says to lowercase this: <https://github.com/pret/pokecrystal/blob/master/STYLE.md#directives>
```suggestion
for BANKNO, NUM_SRAM_BANKS
```